src/server/Session.ts

```diff
@@ -25,3 +27,18 @@
     },
   }
 }
+
+export function closeSessionIfEmpty(session: Session, sessionID: string, sessions: SessionsObject) {
+  if (session.webSockets.size === 0) {
+    console.log(`Session ${sessionID} is now empty. Waiting ${EMPTY_SESSION_TIMEOUT} seconds for potential re-joins before deleting it`)
```

Comment:
This console.log statement is no longer accurate, as the session might have just been created (so it isn't suddenly empty and we aren't waiting for re-joins but rather initial joins). I would probably add an optional parameter to toggle between two messages here.